run-tests.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+: "${NIXPKGS_CHANNEL:=nixpkgs-unstable}"
+
+nix_options=(
+  --no-out-link
+  #--option allow-import-from-derivation false
```

Comment:
Maybe remove disabled options?

Reply:
Fixed in ffa06ac84a5158cf97609f6c5249cdf0dc8b36c1